onchain/rollups/contracts/consensus/authority/Authority.sol

```diff
@@ -43,6 +41,9 @@ contract Authority is AbstractConsensus, Ownable {
     /// @dev MUST be triggered on a successful call to `setHistory`.
     event NewHistory(IHistory history);
 
+    // @notice Raised when a transfer of tokens from an authority to a recipient fails.
```

Comment:
Although errors don't carry NatSpec with them, it's good to prepend documentation comments with `///` (who knows if they make structs and errors support NatSpec in the future?).

Reply:
Addressed in [69c6541](https://github.com/cartesi/rollups/commit/69c65413cc5636e9ec935e851bd549d85352da9b).